src/core/jit/response.rs

```diff
@@ -19,3 +25,13 @@ impl<Value, Error> Response<Value, Error> {
         }
     }
 }
+
+impl Response<async_graphql::Value, jit::Error> {
+    pub fn try_into_hyper_response(self) -> anyhow::Result<hyper::Response<hyper::Body>> {
```

Comment:
Don't need it to be anyhow here, return actual error type.

Reply:
we have it can be either serde or hyper error.. we need anyhow